lib/resque/worker.rb

```diff
@@ -16,6 +16,12 @@ class Worker
 
     WORKER_HEARTBEAT_KEY = "workers:heartbeat"
 
+    @@all_heartbeat_threads = []
```

Comment:
Couldn't you just use `@all_heartbeat_threads` since it will be an instance variable on the class?  Or are you trying to make this kill threads for subclasses if there happen to be any?